ads/google/a4a/traffic-experiments.js

```diff
@@ -323,6 +335,32 @@ export function isInManualExperiment(element) {
 }
 
 /**
+ * Predicate to check whether A4A has launched yet or not, for DoubleClick ads.
+ * If it has not yet launched, then the experimental branch serves A4A, and
+ * control/filler do not. If it has not, then the filler and control branch do
+ * serve A4A, and the experimental branch does not.
+ *
+ * @param {!Element} element  Element to check for pre-launch membership.
+ * @returns {boolean}
+ */
+export function hasLaunchedDoubleClick(element) {
+  return isInExperiment(element, HAS_LAUNCHED_DOUBLECLICK_ID);
+}
+
+/**
+ * Predicate to check whether A4A has launched yet or not, for AdSense ads.
+ * If it has not yet launched, then the experimental branch serves A4A, and
+ * control/filler do not. If it has not, then the filler and control branch do
+ * serve A4A, and the experimental branch does not.
+ *
+ * @param {!Element} element  Element to check for pre-launch membership.
+ * @returns {boolean}
+ */
+export function hasLaunchedAdSense(element) {
+  return isInExperiment(element, HAS_LAUNCHED_ADSENSE_ID);
+}
+
+/**
```

Comment:
Please add tests to `test/test-traffic-experiments.js` to ensure that you're getting the intended behavior in all paths.  (I.e., check the URL-triggered as well as the client-triggered paths.)